designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,49 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+Removing the tab ‘Virtual Machine Templates’ from the side navigation and combining it with ‘Virtual Machines’, so both are located under ‘Virtualization’.
+This PR will focus on how users can create a VM when they choose a to create a VM from a template.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual machine templates tab where they can see the full list of templates.
+From this point, we suggest 2 options to create a VM:
+
+- Create VM via an action from the kebab menu
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+- Create VM via an action from the kebab menu and also allow an action link
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
+
+Both options will lead the user to the general step of the wizard where they will need to provide a name for the VM they want to create.
+
+![UserNeedsToProvideName](img/provide-name.png)
+
+If all they need is to provide a name for the VM, they can skip to the 'Review and create' step where they can review the settings and create the VM.
+
+![user reviews settings](img/CompletesReqiredInfoTemplate.png)
+
+If they need to edit the template before they create the VM, they will need to go through the steps to edit the settings and then create the VM. F.e. if a user wants to change the flavor, they will need to go to the general step and reedit this field.
```

Comment:
They aren't editing the template, but they are starting with a template and then changing the VM settings to go away from the template, right? So maybe phrase this more like "If they need to edit any of the settings for this VM that are different from the template, they will need to go through..."

Reply:
+1